client/my-sites/importer/dispatcher-converter.js

```diff
@@ -0,0 +1,26 @@
+import Dispatcher from 'dispatcher';
+import React from 'react';
+import identity from 'lodash/identity';
+import isFunction from 'lodash/isFunction';
+
+const ifCallableElse = ( f, g ) => isFunction( f ) ? f : g;
```

Comment:
yeah not sure. I wasn't happy with `ifCallableElse` either. still trying to figure out something better…